packs/IDP.json

```diff
@@ -41,7 +121,76 @@
             "traits": [
                 "Maester"
             ],
-            "text": "<b>Challenges Action:</b> Kneel Storm's End Maester to choose an opponent. If you have more power on your faction card than that player, draw ? cards.",
+            "text": "<b>Challenges Action:</b> Kneel Storm's End Maester to choose an opponent. If you have more power on your faction card than that player, draw 1 card.",
+            "deckLimit": 3
+        },
+        {
+            "code": "11088",
+            "type": "attachment",
+            "name": "Glamor",
+            "unique": false,
+            "faction": "baratheon",
+            "loyal": false,
+            "cost": 3,
+            "traits": [
+                "Item",
+                "R'hllor"
+            ],
+            "text": "Character you control only. Shadow (2). Terminal.\nAttached character gets +2 STR.\n<b>Action:</b> Kill attached character to choose a character in your dead pile that was killed this phase. Return the chosen character to play.",
+            "deckLimit": 3
+        },
+        {
+            "code": "11089",
+            "type": "character",
+            "name": "Painted Dogs",
+            "unique": false,
+            "faction": "lannister",
+            "loyal": false,
+            "cost": 4,
+            "icons": {
+                "military": true,
+                "intrigue": true,
+                "power": false
+            },
+            "strength": 3,
+            "traits": [
+                "Clansman"
+            ],
+            "text": "Ambush (4).\n<b>Challenges Action:</b> Return Painted Dogs to your hand to choose and stand a <i>Clansman</i> character or Tyrion Lannister.",
+            "deckLimit": 3
+        },
+        {
+            "code": "11090",
+            "type": "event",
+            "name": "Walk of Atonement",
+            "faction": "lannister",
+            "loyal": true,
+            "cost": 2,
+            "traits": [
+                "The Seven"
+            ],
+            "text": "<b>Dominance Action:</b> Discard any amount of power from a character you control. For each power discarded, discard 1 card at random from each opponent's hand and draw 1 card.",
```

Comment:
There's some italic text on the bottom of the card, do we usually not include that?

Reply:
It's flavor text, we don't include it here and it's a separate field on ThronesDB (similarly, the "Card design by [world champion]" text is a separate field).